server/desktop-api.js

```diff
@@ -54,10 +53,109 @@ function serveTelemetryAPI (app) {
     })
   })
 
-  var basicAuth = auth(secret.credentials)
-  var fileServer = express.static(TELEMETRY_PATH)
-  var indexServer = serveIndex(TELEMETRY_PATH)
-  app.use('/desktop/telemetry', [basicAuth, indexServer, fileServer])
+  app.use('/desktop/telemetry/', [
+    auth(secret.credentials),
+    serveTelemetryDashboard,
+    express.static(TELEMETRY_PATH),
+    express.static(path.join(__dirname, '../static/telemetry-dashboard'))
+  ])
+}
+
+// Summarize telemetry information: active users, monthly growth, most common errors, etc
+function serveTelemetryDashboard (req, res, next) {
+  var isSummary = false
+  if (req.url === '/') {
+    var path = req.originalUrl // eg '/desktop/telemetry'
+    if (!path.endsWith('/')) return res.redirect(path + '/')
```

Comment:
So I can use relative paths in the links.

For example, `<a href="2016-08-01.log">2016-08-01.log</a>`